src/write/segy_write_append.jl

```diff
@@ -5,11 +5,14 @@ function segy_write_append(file::String, block::SeisBlock)
     # Open buffer for writing
     s = open(file, "r+")
 
+    # Write FileHeader
+    if check_fileheader(s, block.fileheader)
```

Comment:
That's not good practice or actually correct. If you use `if` then `check_fileheadert` needs to return true or fals while right now this will return 
```
ERROR: TypeError: non-boolean (Nothing) used in boolean context
```

So you need to have either
- `check_fileheader1` return `True/Flase` and then throw an error based on it
- Have a standalone line `check_fileheader(s, block.fileheader)` that will throw the error i.e:

```
check_fileheader(s, block.fileheader)
# Write Data
ns,ntraces = size(block.data)
for t in 1:ntraces
		write_trace(seekend(s), block, t)
end
```

Reply:
> That's not good practice or actually correct. If you use `if` then `check_fileheadert` needs to return true or fals while right now this will return
> 
> ```
> ERROR: TypeError: non-boolean (Nothing) used in boolean context
> ```
> 
> So you need to have either
> 
> * `check_fileheader1` return `True/Flase` and then throw an error based on it
> * Have a standalone line `check_fileheader(s, block.fileheader)` that will throw the error i.e:
> 
> ```
> check_fileheader(s, block.fileheader)
> # Write Data
> ns,ntraces = size(block.data)
> for t in 1:ntraces
> 		write_trace(seekend(s), block, t)
> end
> ```

Thanks for this comment. I modified it by using a standalone line.